firestore-export-user-data/functions/src/logs.ts

```diff
@@ -0,0 +1 @@
+export const start = () => console.log("Hello, world!");
```

Comment:
remove?

Reply:
Used as a placeholder, more logging will need to be added I guess!